test/quantization/test_quantize_fx.py

```diff
@@ -406,6 +408,122 @@ def forward(self, x):
         FileCheck().check_count('FakeQuantize = prim::GetAttr[name="activation_post_process', 4, exactly=True) \
                    .run(scripted.graph)
 
+    @skipIfNoFBGEMM
+    def test_custom_module_class(self):
+        class CustomModule(torch.nn.Module):
```

Comment:
would there be value in adding a more complicated test case which would only work as a custom class?

Reply:
Also, it will be good to add a test case for dynamic quant with custom modules too.

Reply:
can we use LSTM as a test, I'm not sure what to test for.

Reply:
for dynamic quant test, I'll revisit after the mixed mode support change

Reply:
I think having a complex class in the unittest might be an overkill as it potentially can fail for reasons other than the target of the current unittest. We can instead have multiple unittests for different test scenarios.

Reply:
I think there is value in adding unit tests for the following cases for both static, dynamic and QAT
```python
class CustomModule(torch.nn.Module):
            def __init__(self):
                super().__init__()
                self.conv = torch.nn.LSTM(3, 4)

            def forward(self, x, c):
                return self.lstm(x, c)
```
We are not looking for correct implementation of quantized LSTM module here, just making sure that we can handle multiple inputs and outputs for a realistic use case.

Another test can be test a conv + add as a custom module. In this case,  we can check if the custom module support path produces the same output as the default flow.

Reply:
multiple output is not yet supported in this pr, can be done is a separate pr.
sure I can add conv + add test in a separate pr as well

Reply:
I think it would be better to have a real conv + add use case instead of us guessing about what is needed though